addons/rose/app/styles/rose/components/filter-tags/_index.scss

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.filters-applied {
+  display: flex;
```

Comment:
What happens to the styles when you have a lot of tags that will overflow?

Reply:
It overflows. Thanks for asking, this has now been fixed.